tests/bson/bson-utcdatetime_error-001.phpt

```diff
@@ -4,18 +4,16 @@ BSON BSON\UTCDateTime #001 error
 date.timezone=America/Los_Angeles
 --SKIPIF--
 <?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM handles parameter parsing differently"); ?>
-<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
```

Comment:
Cleanup is only necessary if we're interacting with the database. This simply creates a BSON class.